spec/mercure.md

```diff
@@ -44,165 +45,27 @@ The keywords **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**, **S
 NOT**, **RECOMMENDED**, **MAY**, and **OPTIONAL**, when they appear in this document, are to be
 interpreted as described in [@!RFC2119].
 
-*   Topic: The unit to which one can subscribe to changes. The topic **SHOULD** be identified
-    by an IRI [@!RFC3987]. Using an HTTPS [@!RFC7230] or HTTP [@!RFC7230] URI [@!RFC3986] is
-    **RECOMMENDED**.
+*   Topic: The unit to which one can subscribe to changes. The topic is identified by a string
+    that **MAY** be an IRI [@!RFC3987].
 *   Update: The message containing the updated version of the topic. An update can be marked as
     private, consequently, it must be dispatched only to subscribers allowed to receive it.
-*   Topic selector: An expression matching one or several topics.
+*   Topic matcher: An expression intended to be matched by one or several topics,
+    depending on the matcher type.
+*   Topic matcher type: Several types of matchers **MAY** be supported by the hub. The hub **MUST** support exact matching of topic and **SHOULD** support
+    matching topics using I-Regexp [@!RFC9485], URL patterns [@!urlpattern] and URI Templates [@!RFC6570] as
```

Comment:
I'd say: 

should support at least one of the following topic matchers:

Reply:
I'll improve this by adding a table.